ci/build_only/build.bash

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2018-2020 NVIDIA Corporation.
+# Distributed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+##################################
+# Thrust CPU build script for CI #
+##################################
+
+set -e
+
+# Logger function for build status output
+function logger() {
+  echo -e "\n>>>> ${@}\n"
+}
+
+# Set path and build parallel level
+export PATH=/usr/local/cuda/bin:${PATH}
+
+# Set home to the job's workspace.
+export HOME=${WORKSPACE}
+
+# Switch to project root; also root of repo checkout.
+cd ${WORKSPACE}
+
+# If it's a nightly build, append current YYMMDD to version.
+if [[ "${BUILD_MODE}" = "branch" ]] ; then
+  export VERSION_SUFFIX=`date +%y%m%d`
+fi
+
+# If `CXX` isn't set, assume it's `c++`.
+if [[ -z "${CXX}" ]] ; then
+  export CXX="c++"
+fi
+
+# If `CUDACXX` isn't set, assume it's `nvcc`.
+if [[ -z "${CUDACXX}" ]] ; then
+  export CUDACXX="nvcc"
+fi
+
+CMAKE_FLAGS="-DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_CUDA_COMPILER=${CUDACXX}"
+
+# If it's a nightly build, build all configurations.
+if [[ "${BUILD_MODE}" = "branch" ]] ; then
+  CMAKE_FLAGS="${CMAKE_FLAGS} -DTHRUST_MULTICONFIG_WORKLOAD=FULL"
+fi
```

Comment:
We should get rid of this and just use "c++" and "cu++", which the container will set up.